### PR TITLE
Complete

### PR DESCRIPTION
**Pull request by darshanjayadev**

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+======================*/
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+.container {
+  /* position: relative; */
+  background-image: url("../images/hero.jpg");
```
Comment (oluwoleilesanmi) [inline assets/css/main.css]: Your syntax is correct and works on local machine but on `github previewer` hero.jpg doesnt display.
- Upload hero.jpg image to a desired platform eg [Cloudinary](cloudinary.com)
- Get link to hero.jpg from platform eg [Url](https://res.cloudinary.com/drw0cxkwk/image/upload/v1544707606/Microverse_css_grid_project/weird.jpg)
- Add url to background-image: url([Url](https://res.cloudinary.com/drw0cxkwk/image/upload/v1544707606/Microverse_css_grid_project/weird.jpg));
- Fix any other display issue, push to github, see if it works.

**Review by oluwoleilesanmi — CHANGES_REQUESTED** [review]
Great work @daashandayo @Oghenebrume50 its good you have a reset.css file, clean html and lastly excellent use of relative units of measure %em well done. kindly help make change and you will be good to go. 

- Problem hero.jpg does not display on [Github Previewer](https://htmlpreview.github.io).
- To-recreate paste this [Link](https://github.com/daashandayo/backgrounds-and-gradients/blob/development/index.html) in [Github Previewer](https://htmlpreview.github.io).
- Reason-to-fix not every visitor on your repo will clone the repo to see what it looks like.
- See above on how to attempt fix.

File: assets/css/main.css (hunk 1 above)
Comment (darshanjayadev) [reply assets/css/main.css]: Done

**Review by oluwoleilesanmi — APPROVED** [review]
@daashandayo @Oghenebrume50 well done :+1:

**Pull request merged**